alerter.c

```diff
@@ -1,31 +1,77 @@
 #include <stdio.h>
 #include <assert.h>
 
+typedef enum
+{
+    prod,
+    Test
+} Environment;
 int alertFailureCount = 0;
 
-int networkAlertStub(float celcius) {
-    printf("ALERT: Temperature is %.1f celcius.\n", celcius);
-    // Return 200 for ok
-    // Return 500 for not-ok
-    // stub always succeeds and returns 200
-    return 200;
+int networkAlertStub(float celcius) 

```

Comment:
stub and production code , both are in the same file and as a result compiled code containing both will be part of production software!
There should be possibility to exclude test related code before production code is getting shipped to field.